postgres-appliance/scripts/post_init.sh

```diff
@@ -144,7 +146,8 @@ CREATE TABLE IF NOT EXISTS public.postgres_log (
     application_name text,
     CONSTRAINT postgres_log_check CHECK (false) NO INHERIT
 );
-GRANT SELECT ON public.postgres_log TO admin;"
+GRANT SELECT ON public.postgres_log TO admin;
+RESET search_path;"
```

Comment:
Why do you want to reset it?

IMO, we better use `PGOPTIONS` environment variable to globally enforce `search_path` for all psql connections.
There is already one place where `PGOPTIONS` is used: https://github.com/zalando/spilo/blob/dd7f7181b733ed69317a9ecaa780b5563bac8ee5/postgres-appliance/scripts/post_init.sh#L206

From there it should be moved to the beginning of the script and ` -c search_path=pg_catalog` should be added to the value.